frontend/server/src/CreateUserParams.php

```diff
@@ -57,7 +57,13 @@ class CreateUserParams {
     public $recaptcha = null;
 
     /**
-     * @param array{username?: string, name?: string, email?: string, password?: string, scholar_degree?: string, is_private?: string, gender?: string, recaptcha?: string} $params
+     * @readonly
+     * @var null|int
```

Comment:
Birthdate will be mandatory, so you shouldn't get `null` value
```suggestion
     * @var int
```